org.ryujinx.Ryujinx.yml

```diff
@@ -0,0 +1,30 @@
+app-id: org.ryujinx.Ryujinx
+name: Ryujinx
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: '21.08'
+    directory: lib/ffmpeg
+    add-ld-path: .
+finish-args:
+  # IPC access (required by X11)
+  - --share=ipc
+  # X11 access (TODO: Allows Wayland when EGL is supported correctly on SPB side)
+  - --socket=x11
+  # OpenGL and hidraw (SDL2 controllers) access
+  - --device=all
+  # PulseAudio access
+  - --socket=pulseaudio
+  # Network access (Used by Amiibo API and emulated games)
+  - --share=network
+  # Allows standard filesystem access (Users can load emulated games but also unpack their content)
+  - --filesystem=host
```

Comment:
this is a broad permission.

Reply:
I'm not entirely sure what is the most appropriate permission here considering that we allows people to extract game content to any arbitrary path as said in the comment.

Reply:
I think @hfiguiere is asking to make it opt-in rather than opt-out.

Also, consider implementing [XDG FileChooser](https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.FileChooser) portal.

Reply:
> I think @hfiguiere is asking to make it opt-in rather than opt-out.

In other words, setting the filesystem permission to home and manually letting users allow other paths via e.g. Flatseal?

Reply:
Yes.

Reply:
I see, I will see to change this to only give ``home`` and ``/run/media:ro`` permissions I suppose for now.

> Also, consider implementing [XDG FileChooser](https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.FileChooser) portal if you haven't.

We currently use GTK3 file/directory picker to add new path to scan for games and export game content, would that be enough?

And if GTK3 handles it, do I need to reask for permissions at each boot? (for scan paths registered in the emulator)

Reply:
> We currently use GTK3 file/directory picker to add new path to scan for games and export game content, would that be enough?

It should use that portal without any effort on your part from what I understand. You can try revoking this permission and see if it works.

Reply:
I just confirmed, it works fine without this permission :+1: 
We still do need home:ro for accessing the standard path for console keys